cypress/e2e/48-subworkflow-inputs.cy.ts

```diff
@@ -83,6 +93,21 @@ function navigateWorkflowSelectionDropdown(index: number, expectedText: string)
 		.click();
 }
 
+function populateMapperFields(values: readonly string[], offset: number) {
```

Comment:
Does it make sense to move these local functions to `actions` (maybe [here](https://github.com/n8n-io/n8n/blob/master/cypress/pages/ndv.ts))? I know these pages are getting out of hand (we can maybe create a new one just for RMC), so feel free to push back, but I am worried about mixing actions and utli functions to do the same thing.

Reply:
Yes, strongly agree with this needing a refactor before (maybe master) merge, I'm defining them as I'm iterating for now. I'm also not happy with the sub-functions I'm defining in the big test case.

Reply:
Yes, we have it in few other places, it's not a bad pattern to be honest it's just about consistency. But, looks like a big push to get e2e tests in order is around the corner so good to go like this